resultbrowser/src/js/resultbrowser.js

```diff
@@ -546,8 +546,8 @@
     }
 
     function isJsonContent(contentType) {
-        //  e.g. "application/json" or "application/<...>+json"
-        return /^application\/(.+\+)?json/.test(contentType);
+        //  e.g. "application/json" or "application/<...>+json" or "application/json;charset=utf-8"
+        return /^application\/(.+\+)?json(;.*)?$/.test(contentType);
```

Comment:
Maybe I'm paranoid but better safe than sorry: to ensure that neither leading nor ending whitespace characters break this function, we may trim the input string using our `trim` function declared above. Additionally, the separator character ';' may be preceeded by some whitespace characters as well.

```suggestion
        return /^application\/(.+\+)?json(\s*;.*)?$/.test(trim(contentType));
```